### DISCUSSION TITLE
is there a way to use ChatGPT to read the image?  the accuracy is much higher, you don't need to have ROI. Just call chatGPT API and upload an image.

### DISCUSSION THREAD

**Discussion by goblueapple** [jomjol/AI-on-the-edge-device#3198] [General]
### The Feature

the accuracy is much higher, you don't need to have ROI. Just call chatGPT API and upload an image. Is that possible to do that in ESP32? anyone has done that?
![image](https://github.com/jomjol/AI-on-the-edge-device/assets/16480565/85f6a427-6171-4a7a-a006-26f67883e60b)

**Comment by SybexX** [comment]
you can find the last picture below "http:// your IP /img_tmp/raw.jpg"
The value read is wrong, it should be 4331.365
For ChatGPT you just need a simple esp32-cam-webserver.

e.g. like this https://github.com/easytarget/esp32-cam-webserver
